Wootric/WootricSDK-Android
# Bug: fix font size

Fix survey rating numbers hidden when device is set to maximum font size

## Test

Notes: changing the device's font size will depend on the device. I recommend testing with different devices, specially smaller devices where this bug is more noticeable

- Go to Settings -> Display -> Advanced -> Font Size and set it to Largest
- Set up everything to show the survey. The rating numbers should all be visible.
- Try with different Font sizes
- Try with different modes (NPS, CSAT, CES)
